vcd/resource_vcd_lb_service_monitor_test.go

```diff
@@ -103,7 +103,7 @@ func testAccCheckVcdLbServiceMonitorDestroy(serviceMonitorName string) resource.
 			return fmt.Errorf(errorUnableToFindEdgeGateway, err)
 		}
 
-		monitor, err := edgeGateway.ReadLBServiceMonitor(&types.LBMonitor{Name: serviceMonitorName})
+		monitor, err := edgeGateway.GetLbServiceMonitor(&types.LbMonitor{Name: serviceMonitorName})
```

Comment:
Doesn't we should have wappers here?

Reply:
Yes indeed. I was hoping `make install` to fail on such cases but I was wrong. Replaced them.